recipes/gobject-introspection/all/conanfile.py

```diff
@@ -110,3 +109,19 @@ def package_info(self):
         bin_path = os.path.join(self.package_folder, "bin")
         self.output.info("Appending PATH env var with: {}".format(bin_path))
         self.env_info.PATH.append(bin_path)
+
+        exe_ext = ".exe" if self.settings.os == "Windows" else ""
+
+        pkgconfig_variables = {
+            'datadir': '${prefix}/res',
+            'bindir': '${prefix}/bin',
+            'g_ir_scanner': '${bindir}/g-ir-scanner',
```

Comment:
What if we're cross building?
Will these executables run when I'm building something for another os/arch?